profiles/common/modules/custom/tmgmt_poetry/plugins/tmgmt_poetry.plugin.inc

```diff
@@ -86,23 +90,31 @@ class TMGMTPoetryTranslatorPluginController extends TMGMTDefaultTranslatorPlugin
     $item = array_shift($items);
     if (!empty($item)) {
       $results = db_select('tmgmt_job_item', 'tji')
-          ->fields('tji', array('tjid'))
-          ->condition('item_type', $item->item_type, '=')
-          ->condition('item_id', $item->item_id, '=')
-          ->execute()
-          ->fetchAllAssoc('tjid');
+        ->fields('tji', array('tjid'))
+        ->fields('tji', array('tjiid'))
```

Comment:
> otherwise only tjiid will be present after execution (suppose to be)

That's not how it's supposed to be. fields() adds the fields, it does not remove any previous selection of fields. So our code actually works as expected. See https://api.drupal.org/api/drupal/includes!database!select.inc/function/SelectQuery%3A%3Afields/7.x.

For the clarity of the code I decided to merge them though as you suggested.